task-tracker-webapp/src/main/java/com/isa/tasktrackerwebapp/controller/TaskController.java

```diff
@@ -0,0 +1,31 @@
+package com.isa.tasktrackerwebapp.controller;
+
+import com.isa.tasktrackerwebapp.model.JsonDataTask;
+import com.isa.tasktrackerwebapp.model.Task;
+import com.isa.tasktrackerwebapp.service.TaskService;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.util.List;
+
+@Controller
+public class TaskController {
+    @GetMapping("/taskList")
+    public String taskList(Model model, @RequestParam(required = false) String sortBy, @RequestParam(required = false) String searchTaskName, @RequestParam(required = false) String filterActive) {
+        List<Task> taskList = JsonDataTask.getTasks();
+
+        taskList = TaskService.howToSortTask(sortBy, taskList);
+
+        taskList = TaskService.filterTasksByName(taskList, searchTaskName);
+
+        taskList = TaskService.filterTasksByActive(taskList, filterActive);
+
```

Comment:
to bym ukrył w jakimś serwisie, który jest tutaj wstrzyknięty jako bean i wtedy te metody nie będą musiały być statyczne no i też czy nie dałoby się tego filtrowania i sortowania zrobić w 1 metodzie?